Repairs/app/src/main/java/com/rooio/repairs/ChooseEquipmentAdapter.kt

```diff
@@ -0,0 +1,206 @@
+package com.rooio.repairs
+
+import android.content.Context
+import android.graphics.Color
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.*
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.recyclerview.widget.RecyclerView
+import androidx.transition.TransitionManager
+import java.util.*
+import kotlin.collections.ArrayList
+
+class ChooseEquipmentAdapter(context: Context, dataList: ArrayList<EquipmentData>) : RecyclerView.Adapter<ChooseEquipmentAdapter.ViewHolder>() {
+
+    private val equipmentDataList = dataList
+    private val equipmentData: ArrayList<EquipmentData> = ArrayList()
+    private val locations: ArrayList<String> = ArrayList()
+    private val applicationContext = context
+
+    init {
+        this.equipmentData.addAll(equipmentDataList)
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        val view =
+                LayoutInflater.from(applicationContext).inflate(R.layout.equipment_item, parent, false)
+        return ViewHolder(view)
+    }
+
+    private fun configureElements(holder: ViewHolder, position: Int){
+        holder.equipmentView.setOnClickListener{null}
+        holder.equipmentName.setTextColor(Color.parseColor("#333232"))
+        holder.equipmentLayout.setBackgroundResource(R.drawable.gray_button_border)
+        holder.grayDropDown.visibility = View.VISIBLE
+
+        if (position == 0) {
+            holder.locationDivider.visibility = View.GONE
+        }
+        if (equipmentDataList[position].name == "General HVAC (No Appliance)" ||
```

Comment:
maybe to simplify this, just search for the word "general" in equipmentDataList[position].name.

that way there's not 3 || statements

Reply:
Great idea! Only worry I have is the unlikely case someone adds an equipment piece with the word "General" inside of it?